src/lib/web-worker/worker-exec.ts

```diff
@@ -27,7 +27,8 @@ export const initNextScriptsInWebWorker = async (initScript: InitializeScriptDat
   let errorMsg = '';
   let env = environments[winId];
   let rsp: Response;
-
+  let contentTypeBlocklist = ["image/gif"]
```

Comment:
I think it's a good idea to validate that's javascript before we execute it. But could we do the opposite here, where we have a whitelist of content types which are valid javascript?

Reply:
My concern with using a whitelist was that maybe some request doesnt return the correct contentType  even though they are actually javascripts. If that is not something that you guys think could happen I can implement it as a white list :)

Reply:
Done